website_rma/views/request_rma_form.xml

```diff
@@ -0,0 +1,160 @@
+                                            </div>
+                                        </div>
+                                    </div>
+                                </section>
+                            </div>
+                            <div class="request_rma_container">
+                                <form action="/website_form/"
+                                      method="post"
+                                      data-model_name="rma"
+                                      data-success_page="/requestrma-thank-you"
+                                      class="s_website_form container-fluid mt32"
+                                      enctype="multipart/form-data"
+                                      data-editable-form="false"
+                                      id="rma_request_form">
+                                    <div class="form-group row">
+                                        <label class="col-lg-3 col-md-4 col-form-label"
+                                               for="product_id">
+                                            Product
+                                        </label>
+                                        <div class="col-lg-7 col-md-8">
+                                            <input type="hidden"
+                                                   name="product_id"
+                                                   class="form-control o_website_form_input"/>
+                                        </div>
+                                    </div>
+                                    <div class="form-group row form-field o_website_form_required">
+                                        <label class="col-lg-3 col-md-4 col-form-label"
+                                               for="product_uom_qty">
+                                            Quantity
+                                        </label>
+                                        <div class="col-lg-4 col-md-5">
+                                            <input class="form-control o_website_form_input"
+                                                   pattern="^\d*(\.\d{0,3})?$"
+                                                   name="product_uom_qty"
+                                                   required=""
+                                                   t-att-value="request.params.get('product_uom_qty', '1.000')"/>
+                                        </div>
+                                        <div class="col-lg-3 col-md-3">
+                                            <t t-set="uom_unit_id" t-value="request.env.ref('uom.product_uom_unit').id"/>
+                                            <select name="product_uom" class="form-control custom-select o_website_form_input">
+                                                <option/>
+                                                <t t-foreach="request.env['uom.uom'].sudo().search([])" t-as="uom">
```

Comment:
I find this a little bit missleading, to be honest, as it throws all the uom availble and it can lead to errors. Maybe it would be better to just put the selected product default sale UoM

![image](https://user-images.githubusercontent.com/5040182/93485437-5be25880-f903-11ea-923c-d85f449cfc0e.png)